controllers/fileController.js

```diff
@@ -0,0 +1,14 @@
+const FacadeFunctions = require('../facade/facadeFunctions');
+
+module.exports = [
+    {
+        path: '/files',
+        method: 'POST',
+        handler: FacadeFunctions.upload
+    },
+    {
+        path: '/files/{id}',
+        method: 'GET',
+        handler: FacadeFunctions.download
+    }
+];
```

Comment:
In controllers, both fileController and issueController you are still defining routes. Here you should export functions that will be used in routes modules:

FileController.create

This means that fileController needs to export create function. Just like you did with facade.